### PR TITLE
Update 0061-allow-custom-task-to-be-embedded-in-pipeline.md

### PR DESCRIPTION
**Pull request by ScrapCodes**

### CODE REVIEW

File: teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md [hunk 1]
```diff
@@ -73,8 +74,8 @@ See issue [tektoncd/pipeline#3682](https://github.com/tektoncd/pipeline/issues/3
 
 ### Non-Goals
 
-1. Custom task controllers are to be developed by other parties. Custom task
-specification should not be validated by Tektoncd/Pipeline webhooks.
+1. Custom task controllers are developed by parties. Whether a custom task
```
Comment (ScrapCodes) [inline teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md:77]: Note to self, re phrase this.

**Comment by ScrapCodes** [comment]
@Tomcli Can you please take a look, if these changes look ok.

File: teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md [hunk 2]
```diff
@@ -37,48 +37,51 @@ authors:
 
 Tektoncd/Pipeline currently allows custom task to be referenced in pipeline
 resource specification file using [`taskRef`](https://github.com/tektoncd/community/blob/main/teps/0002-custom-tasks.md).
-That feature allows a custom task reference to be submitted to kubernetes along
+This TEP discusses the various aspects of embedding the custom task in the `RunSpec`. Just as a regular task, can be either referenced
```
Comment (Tomcli) [inline teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md:40]:
```suggestion
This TEP discusses the various aspects of embedding the custom task in the `TaskSpec` for Tekton Pipeline CRD and `RunSpec` for Tekton Run CRD. Just as a regular task, can be either referenced
```

File: teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md [hunk 3]
```diff
@@ -37,48 +37,51 @@ authors:
...
+or embedded in the `pipelineRun`, after implementation of this TEP, a simillar support will be available for custom task as well.
```
Comment (Tomcli) [inline teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md:41]:
```suggestion
or embedded in the `pipelineRun`, after implementation of this TEP, a similar support will be available for custom task controller as well.
```

File: teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md [hunk 4]
```diff
@@ -37,48 +37,51 @@ authors:
...
+
+## Motivation
+
+A custom task reference needs to be submitted to kubernetes along with
 the submission of the [Tektoncd/pipeline](https://github.com/tektoncd/pipeline).
-However, to run the pipeline, custom task CRD need to be submitted separately as separate requests to Kubernetes.
-If multiple custom task CRDs are created with the same name, to both Kubernetes and Tektoncd/Pipeline,
+To run the pipeline, custom task resource object creation is submitted as a separate requests to Kubernetes.
```
Comment (Tomcli) [inline teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md:47]:
```suggestion
To run the pipeline, custom task resource object creation is submitted as a separate request to Kubernetes.
```

File: teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md [hunk 5]
```diff
@@ -37,48 +37,51 @@ authors:
...
+then the behavior will be unified with regular task, retaining the existing the behavior of `taskRef`. 
+The embedding of spec avoids the issues related to naming conflict, when multiple users in the
+same namespace create resource. Related issue 
+[tektoncd/pipeline#3682](https://github.com/tektoncd/pipeline/issues/3682)
 
 ### Goals
 
 1. Allow custom tasks to be embedded in a pipeline specification.
 2. Custom taskSpec should be submitted as part of the runSpec.
+3. Document, general advice on validation/verification of custom task, to the custom task developers.
```
Comment (Tomcli) [inline teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md:79]:
```suggestion
3. Document, general advice on validation/verification of custom task, to the custom task controller developers.
```

**Review by Tomcli — COMMENTED** [review]
thanks @ScrapCodes. I added few comments on some of the wording. Overall it looks good to me.

**Pull request merged**